src/components/application/application.js

```diff
@@ -19,6 +25,15 @@ function Application({ logout, user = {} }) {
       </header>
       <main className="main">
         <h1>Dashboard</h1>
+
+        {route === "form"
```

Comment:
I think your suggestion is spot on, the whole managing of a 'route' here is non-intuitive. If it is a genuine route change we should manage this with react-router or similar. But as you suggest this could probably just be managed inside a single component for now.